doc/source/examples/unresolved-cfd-dem/liquid-solid-fluidized-bed/liquid-solid-fluidized-bed.rst

```diff
@@ -0,0 +1,500 @@
+        set poisson ratio particles           = 0.3
+        set restitution coefficient particles = 0.9
+        set friction coefficient particles    = 0.1
+        set rolling friction particles        = 0.2
+      end
+      set young modulus wall           = 1e7
+      set poisson ratio wall           = 0.3
+      set restitution coefficient wall = 0.2
+      set friction coefficient wall    = 0.1
+      set rolling friction wall        = 0.3
+    end
+
+The amount of particles used for alginate particles is 107960.
+    
+Insertion Info
+~~~~~~~~~~~~~~~~~~~
+
+The volume of the insertion box should be large enough to fit all particles. Also, its bounds should be located within the mesh generated in the Mesh subsection.
+
+.. code-block:: text
+
+    subsection insertion info
+      set insertion method                               = non_uniform
+      set inserted number of particles at each time step = 7240
+      set insertion frequency                            = 100000
+      set insertion box minimum x                        = 0
+      set insertion box minimum y                        = -0.030
+      set insertion box minimum z                        = -0.030
+      set insertion box maximum x                        = 0.53
+      set insertion box maximum y                        = 0.030
+      set insertion box maximum z                        = 0.030
+      set insertion distance threshold                   = 1.8
+      set insertion random number range                  = 0.3
+      set insertion random number seed                   = 19
+    end
+
+.. note::
+    At each time-step, 7240 particles are inserted. This means that, to reach the full amount of particles (72400), 10 insertion time steps are needed. Additionally, particles need to be fully settled before the fluid injection. Hence, ``time end`` in ``subsection simulation control`` needs to be chosen accordingly.
+
+
+---------------------------
+Running the DEM Simulation
+---------------------------
+Launching the simulation is as simple as specifying the executable name and the parameter file. Assuming that the ``dem`` executable is within your path, the simulation can be launched on a single processor by typing:
+
+.. code-block:: text
+
+  dem dem-packing-in-fluidized-bed.prm
+
+or in parallel (where 8 represents the number of processors)
+
+.. code-block:: text
+
+  mpirun -np 8 dem dem-packing-in-fluidized-bed.prm
+
+Lethe will generate a number of files. The most important one bears the extension ``.pvd``. It can be read by popular visualization programs such as `Paraview <https://www.paraview.org/>`_. 
+
+
+.. note:: 
+    Running the packing of alumina particles should take approximately 4 hours and 15 minutes on 16 cores. For the alginate particles, it takes approximately 5 hours and 52 minutes (``time end = 9.9``).
+
+After the particles have been packed inside the cylinder, it is now possible to simulate the fluidization of particles.
```

Comment:
Do you really need 10 seconds to pack the cylinder? Seems like a lot!
Are you sure it takes this much time?

Reply:
The insertion parameters are very outdated. I just optimized it. I am waiting on Narval, but 2.5 s should be enough with the new parameters.